src/util/ActiveShapeUtils.tsx

```diff
@@ -105,151 +103,3 @@ export function Shape<OptionType, ExtraProps, ShapePropsType>({
-  return _item != null && 'sectors' in graphicalItem.props;
-}
-
-export function isScatter(graphicalItem: GraphicalItem, _item: unknown): _item is ScatterItem {
-  return _item != null && 'points' in graphicalItem.props;
-}
-
-export function compareFunnel(shapeData: FunnelItem, activeTooltipItem: FunnelItem) {
-  const xMatches = shapeData.x === activeTooltipItem?.labelViewBox?.x || shapeData.x === activeTooltipItem.x;
-  const yMatches = shapeData.y === activeTooltipItem?.labelViewBox?.y || shapeData.y === activeTooltipItem.y;
-  return xMatches && yMatches;
-}
-
-export function comparePie(shapeData: PieItem, activeTooltipItem: PieItem) {
-  const startAngleMatches = shapeData.endAngle === activeTooltipItem.endAngle;
-  const endAngleMatches = shapeData.startAngle === activeTooltipItem.startAngle;
-  return startAngleMatches && endAngleMatches;
-}
-
-export function compareScatter(shapeData: ScatterItem, activeTooltipItem: ScatterItem) {
-  const xMatches = shapeData.x === activeTooltipItem.x;
-  const yMatches = shapeData.y === activeTooltipItem.y;
-  const zMatches = shapeData.z === activeTooltipItem.z;
-  return xMatches && yMatches && zMatches;
-}
-
-function getComparisonFn(graphicalItem: GraphicalItem, activeItem: unknown) {
-  let comparison: (shapeData: unknown, activeTooltipData: unknown) => boolean;
-
-  if (isFunnel(graphicalItem, activeItem)) {
-    comparison = compareFunnel;
-  } else if (isPie(graphicalItem, activeItem)) {
-    comparison = comparePie;
-  } else if (isScatter(graphicalItem, activeItem)) {
-    comparison = compareScatter;
-  }
-
-  return comparison;
-}
-
-function getShapeDataKey(graphicalItem: GraphicalItem, activeItem: unknown): GraphicalItemShapeKey {
-  let shapeKey: GraphicalItemShapeKey;
-
-  if (isFunnel(graphicalItem, activeItem)) {
-    shapeKey = 'trapezoids';
-  } else if (isPie(graphicalItem, activeItem)) {
-    shapeKey = 'sectors';
-  } else if (isScatter(graphicalItem, activeItem)) {
-    shapeKey = 'points';
-  }
-
-  return shapeKey;
-}
-
-function getActiveShapeTooltipPayload(graphicalItem: GraphicalItem, activeItem: unknown): ShapeData {
-  if (isFunnel(graphicalItem, activeItem)) {
-    return activeItem.tooltipPayload?.[0]?.payload?.payload;
-  }
-  if (isPie(graphicalItem, activeItem)) {
-    return activeItem.tooltipPayload?.[0]?.payload?.payload;
-  }
-
-  if (isScatter(graphicalItem, activeItem)) {
-    return activeItem.payload;
-  }
-
-  return {} as ShapeData;
-}
-/**
- *
- * @param {GetActiveShapeIndexForTooltip} arg an object of incoming attributes from Tooltip
- * @returns {number}
- *
- * To handle possible duplicates in the data set,
- * match both the data value of the active item to a data value on a graph item,
- * and match the mouse coordinates of the active item to the coordinates of in a particular components shape data.
- * This assumes equal lengths of shape objects to data items.
- */
-export function getActiveShapeIndexForTooltip({
```

Comment:
everything for active shape still works correctly even without all of this? I feel like we added this recently?

Reply:
All active shapes I'm aware of are working in storybook.

Reply:
ack, cool